src/service/performance-impl.js

```diff
@@ -302,6 +302,13 @@ export class Performance {
         // Tick the "messaging ready" signal.
         this.tickDelta('msr', this.win.Date.now() - this.initTime_);
 
+        // Tick timeOrigin so that epoch time can be calculated by consumers.
+        this.tickDelta(
+          'timeOrigin',
+          this.win.performance.timeOrigin ||
+            this.win.performance.timing.navigationStart
```

Comment:
I think we also need to make all our `initTime_`'s relative to `timeOrigin`, too. Or, we can just emit a `timeOrigin` tick using `initTime`.

Reply:
You are right, but it doesn't need to happen in this PR. We are already inconsistent and this PR leaves us at the same level of inconsistency.

- browser given deltas are all in terms of timeOrigin already
- v0 calculated deltas (e.g. 'pc', 'ol', etc.) are relative to `initTime_`

This PR doesn't change that inconsistency, although I do agree that we should update it as part of https://github.com/ampproject/amphtml/issues/16042